website/docs/basics/blocks-component-in-block.md

```diff
@@ -41,8 +114,9 @@ and you want to use the heading component in the Jumbotron block, you can use it
 `src/Blocks/custom/jumbotron/manifest.json`
 
 ```json
+"blockName": "jumbotron",
```

Comment:
On line 112 there is a typo `component's` -> `component`

Reply:
fixed